### PR TITLE
Update to 0.5.7

### PR DESCRIPTION
**Pull request by Alecaddd**

### CODE REVIEW

File: com.github.alecaddd.sequeler.json [hunk 1]
```diff
@@ -178,7 +178,7 @@
       "config-opts": ["--buildtype=release"],
       "sources": [{
         "type": "archive",
-        "url": "https://github.com/Alecaddd/sequeler/archive/v0.5.5.tar.gz",
+        "url": "https://github.com/Alecaddd/sequeler/archive/v0.5.7.tar.gz",
         "sha256": "ac5cd238bffe3afaf1c931009f7737cf0f22ef072a5b6b977149b38f4f20782b"
```
Comment (bilelmoussaoui) [inline com.github.alecaddd.sequeler.json]: You should update the sha256 too

Comment (Alecaddd) [reply com.github.alecaddd.sequeler.json]: Thanks, how do I do that? Where do I grab that value? 😅

Comment (nedrichards) [reply com.github.alecaddd.sequeler.json]: Run `sha256sum $YOUR_FILENAME` on the command line and copy the value.

**Comment by nedrichards** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by nedrichards** [comment]
Builds fine: https://flathub.org/builds/#/builders/2/builds/2797

**Pull request merged**